docs/source/data/subscriptions.mdx

```diff
@@ -698,54 +700,57 @@ Use [`WebSocketLink`](/react/api/link/apollo-link-ws/)
 
 ### Switching to `graphql-ws`
```

Comment:
Perhaps this "migrating to graphql-ws" part could be removed and we can link to the v3 docs instead.

Reply:
Love this!